File: src/main/java/io/github/et/subprocessLoader/MCServer.java
package io.github.et.subprocessLoader;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class MCServer {
    public String name;
    public String workingDir;
    public String command;
    public long group;
    public boolean useAutoBackUp;
    public boolean useAdvancement;
    public boolean useDeathMsg;
    public MCServer(String name,String workingDir,String command, long group, boolean useAutoBackUp,boolean useDeathMsg, boolean useAdvancement){
        this.name = name;
        this.workingDir = workingDir;
        this.command = command;
        this.group = group;
        this.useAutoBackUp = useAutoBackUp;
        this.useDeathMsg = useDeathMsg;
        this.useAdvancement=useAdvancement;
    }
}

File: SubProcess/src/main/kotlin/io/github/et/MCServer.kt
package io.github.et


class MCServer(
    val name: String,
    val workingDir: String,
    val command: String,
    val group: Long,
    val useAutoBackup: Boolean,
    val useDeathMessage:Boolean,
    val useAdvancement: Boolean,
    var encoding: String ="UTF-8"
)
